test/integration/objectstore/test_objectstore_datatype_upload.py

```diff
@@ -14,17 +14,27 @@
     UploadTestDatatypeDataTestCase,
 )
 
+REFRESH_TIME = 30
+CONNECTION_POOL_MONITOR_INTERVAL = 60
```

Comment:
Can you make these much shorter please ? Ideally less than a second

Reply:
Reduced these to 3 and 6 seconds (instead of 30 and 60). Can't reduce to it to less than a second, as must allow the upload to start/complete before marking a connection stale. This reduction may make the test more flaky. We'll see, I guess.